docs/UserGuide.md

```diff
@@ -164,9 +164,8 @@ Examples:
 
 :information_source:
  * The default value for `PROTEIN`, `FATS` and `CARBS` is 0. The default date is the day in which the food item is added.<br>
- * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer.<br>
+ * Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer that is smaller than 2^31.<br>
```

Comment:
It must be smaller than 1000 rather than 2^31. Same for the ones below

Reply:
> Your implementation is q clean too without many repeats but it involves the creation of a Macronutrient when the value is needed.
> 
> It might be better to have a static reference to a default macronutrient to be accessed instead of creating a new one to be accessed everytime

You got a point here. I've changed it back to the original one